install.py

```diff
@@ -95,14 +105,16 @@ def check_distribution_compatibility():
 
 
 def get_distribution_info():
-	# return distribution name and major version
-	if platform.system() == "Linux":
-		current_dist = platform.dist()
-		return current_dist[0].lower(), current_dist[1].rsplit('.')[0]
-
-	elif platform.system() == "Darwin":
-		current_dist = platform.mac_ver()
-		return "macos", current_dist[0].rsplit('.', 1)[0]
+
+    # return distribution name and major version
+
+	if platform.system() == 'Linux':
+            return (distro.id(), distro.major_version())
+	
+        elif platform.system() == 'Darwin':
+
+            current_dist = platform.mac_ver()
+            return ('macos', current_dist[0].rsplit('.', 1)[0])
```

Comment:
Indentations are off here. Use tabs only and remove unnecessary new-lines on L108, L110 and L115